login/login.js

```diff
@@ -29,8 +29,8 @@ async function checkUserCredentials(userData) {
   };
   let status = StatusCodes.StatusCodes.OK;
 
-  const { rows: userDb } = await db.queryWithParams('SELECT * FROM "user" where email=$1', [userData.email]);
-
+  const dBName = constants.CONSTANTS.DATABASE === "Postgree" ? '"user"' : "[user]";
```

Comment:
Not clear name. 'User' is not a DB this is Table.

Reply:
fixed